google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.imageURI);
```

Comment:
Just tried with the computed property. I like it as a better approach in general, but it always re-calculates, even if there are no bindings on it. I don't know if it's really possible to do this without incurring the computation cost at draw time.